src/App.jsx

```diff
@@ -28,6 +28,12 @@ export default function App() {
       localStorage.setItem("currentTaskName", JSON.stringify(taskName));
   }, [taskName]);
 
+  useEffect(() => {
+    const filteredByCurrentName = tasks.filter(task => task.name === taskName);
+
+    if (filteredByCurrentName.length === 0) return setTaskName("Time to focus!")
+  }, [setTaskName, taskName, tasks])
+
```

Comment:
It doesn't work properly :thinking: 
When one task is selected:

![image](https://github.com/user-attachments/assets/e7788c82-47dd-4bf5-a8d0-065ef0873c56)

and we try to delete other task (without selecting it):

![image](https://github.com/user-attachments/assets/ea8bf75f-97e4-4d83-9ae0-514284bcbfec)

selection disappear:

![image](https://github.com/user-attachments/assets/9f987d80-c0f5-4ad5-be3d-7d8070da5d36)

Reply:
Separate issue for this problem: https://github.com/wilkus27/pomodoro-soup/issues/8